src/commands/synthetics/utils/internal.ts

```diff
@@ -87,3 +87,106 @@ export const toExecutionRule = (env: string | undefined): ExecutionRule | undefi
 
   return undefined
 }
+
+type AccumulatorBaseConfigOverride = Omit<
+  UserConfigOverride,
+  | 'retry'
+  // TODO SYNTH-12971: These options will be implemented later in separate PRs
+  | 'basicAuth'
+  | 'headers'
+  | 'cookies'
+  | 'deviceIds'
+  | 'locations'
+  | 'mobileApplicationVersion'
+  | 'mobileApplicationVersionFilePath'
```

Comment:
These are simple strings, so I think you can implement them in this PR. This was overlooked in the Jira card